requirements/travis.txt

```diff
@@ -2,17 +2,29 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements/travis.txt requirements/travis.in
```

Comment:
Can you add the following in the `Makefile` to avoid getting ^^ message in each `.txt` file?
`upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade`

Reply:
ok